### PR TITLE
Run detekt CI with JDK 12

### PR DESCRIPTION
**Pull request by schalkms**

### CODE REVIEW

File: .github/workflows/pre-merge.yaml [hunk 1]
```diff
@@ -14,7 +14,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
-        jdk: [8, 11, 13]
+        jdk: [8, 11, 12]
```
Comment (cortinico) [inline .github/workflows/pre-merge.yaml:17]:
```suggestion
        jdk: [8, 11, 12, 13]
```
How about this?

**Comment by schalkms** [comment]
This was a test drive because of #2681

**Comment by schalkms** [comment]
I agree, we don't want to support each combination of JDKs, Gradle versions and so on...
The CI passes, that's why I think #2681 is a not a problem related to detekt.

**Comment by cortinico** [comment]
My bad, I haven't realised this was a test run

**Comment by schalkms** [comment]
@cortinico I could have written a notice in the description.

**Pull request closed without merging**